recipes/nomad/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+export GO111MODULE=on
+make deps
+
+# XXX We should run "make prerelease" here instead but that fails for some reason
```

Comment:
@xhochy want to try `make prelease`?

Reply:
This isn't working as a lot of protobuf sources aren't part of the tarball. From the looks, the generated files were probably generated as part of the tarball creation process. Thus I think the commands here look correct.